### ISSUE TITLE
provide option to specify ca_file

### ISSUE THREAD

**Issue by juliantaylor** [kawasakitoshiya/fluent-plugin-out-http-ext#25]
The plugin hardcodes OpenSSL::X509::DEFAULT_CERT_FILE as the ca file.
https://github.com/toshitanian/fluent-plugin-out-http-ext/blob/master/lib/fluent/plugin/out_http_ext.rb#L220
Unfortunately on e.g. Debian this variable points to a non existent file, the system ca bundle is `/etc/ssl/certs/ca-certificates.crt`

Setting the SSL_CERT_FILE variable also does not appear to override this ruby variable (https://www.openssl.org/docs/man1.1.1/man3/SSL_CTX_set_default_verify_paths.html)

Please provide a configuration option to set the ca_file to use manually.

**Comment by cosmo0920** [comment]
Hi, [upstream plugin](https://github.com/fluent-plugins-nursery/fluent-plugin-out-http) supports ca_file option.
Is it enough for you?

**Comment by juliantaylor** [comment]
yes the updated fluentd plugin provides the feature now, if your code is based on that this issue can be closed.
